client/blocks/upwork-banner/index.jsx

```diff
@@ -12,28 +12,35 @@ class UpworkBanner extends PureComponent {
 	static propTypes = {
 		currentPlan: PropTypes.object,
 		location: PropTypes.string.isRequired,
+		utmCampaign: PropTypes.string.isRequired,
 		siteId: PropTypes.number,
 		translate: PropTypes.func.isRequired,
 	};
 
 	render() {
-		const { translate, location, currentPlan } = this.props;
+		const { translate, location, utmCampaign, currentPlan } = this.props;
 		const plan = currentPlan?.productSlug;
+		const queryString = new URLSearchParams( {
+			utm_campaign: utmCampaign,
+			utm_medium: 'automattic_referred',
+			utm_source: 'WordPresscom',
+		} ).toString();
+		const builtByWpUrl = `https://builtbywp.com/?${ queryString }`;
```

Comment:
This can be replaced completely with the URL native object, (window.open supports it :))

```javascript

const builtByWpUrl = new URL('https://builtbywp.com/');
builtByWpUrl.search = new URLSearchParams( {
			utm_campaign: utmCampaign,
			utm_medium: 'automattic_referred',
			utm_source: 'WordPresscom',
} )

Reply:
TIL! Updated in https://github.com/Automattic/wp-calypso/pull/68138/commits/2a09f9d4db5926177c2da27b7196cdbf90c257b8